### ISSUE TITLE
WPA network connections only work through saved connections

### ISSUE THREAD

**Issue by buzztiaan** [maemo-leste/bugtracker#386]
On being around a new AP, if i just select the wifi network in the list and supply the connections preshared key in the popup that follows, it never is able to connect.

Adding the AP through settings > internet > connections > new, etc -does- connect totally fine.

**Comment by parazyd** [comment]
This has happened to me recently too on the N900.

**Comment by MerlijnWajer** [comment]
```
20:52 < Wizzup> It works depending on how fast you type
20:52 < Wizzup> The problem is that, I think, if you don't do it before a new scan completes, the plugin loses the matching with wpasupplicant
```

**Comment by buzztiaan** [comment]
yeah ok, on trying to reproduce, the connection issue remained present with the AP saved. it took a couple retries before it would connect.

i'm no longer convinced the connection issues are based on just the saved connections anymore, shall i close the issue?
